config/storybook-preset-carbon/package.json

```diff
@@ -17,10 +17,15 @@
     "components",
     "react"
   ],
+  "scripts": {
+    "build": "babel ./carbon-theme-addon/src --out-dir ./dist"
```

Comment:
Can you add a `clean` task that removes the built files? In other packages we've been using `rimraf`, e.g.:

```json
{
  "scripts": {
     "clean": "rimraf dist"
  }
}
```